test/colors_svg_test.dart

```diff
@@ -10,7 +10,7 @@ void main() {
     expect(parseColor('#FFFFFF'), white);
     expect(parseColor('white'), white);
     expect(parseColor('rgb(255, 255, 255)'), white);
-    expect(parseColor('rgb(100%, 100%, 100%)'), white);
+    expect(parseColor('RGB(  100%   ,   100.0% ,  99.9999% )'), white);
```

Comment:
This is fine, but we might as well keep the original assertion in there as well.